webui/react/src/pages/TrialDetails/LogViewer.tsx

```diff
@@ -0,0 +1,527 @@
+import Button from 'hew/Button';
```

Comment:
i think maybe we should add this (highlighting a selected log) in hew instead of copying the component out of it with modifications. it'd also make reviewing the actual changes easier.

Reply:
We ended up changing a bunch of `LogViewer` including moving the action buttons out of the component and moving the `logs` state out, I'm not sure it's worth it to have all though flags to distinguish.. If we want to move the component to hew, we could extract some common functions and have two `LogViewer` components?

Reply:
Sounds like a plan, but since I also changed some hooks, the component could get complicated. 
I want to propose to leave the updated `LogViewer` in determined repo for now, since this is a new feature and in the next couple releases we may need to address feedbacks/bugs, once everything becomes stable, we can go ahead to either merge or migrate the previous `LogViewer`. Does this sound good to you?

Reply:
sounds good -- just link a ticket in this pr so we can track extracting this.